docusaurus/docs/create-a-plugin/develop-a-plugin/build-a-panel-plugin-with-d3.md

```diff
@@ -14,52 +14,59 @@ keywords:
   - panel plugin
 ---
 
+import CreatePlugin from '@shared/create-plugin-frontend.md';
+
 ## Introduction
 
-Panels are the building blocks of Grafana, and allow you to visualize data in different ways. This tutorial gives you a hands-on walkthrough of creating your own panel using [D3.js](https://d3js.org/).
+Panels allow you to visualize data in different ways; they are one of the fundamental building blocks of Grafana. Grafana has several types of panels already built-in, and many more available in the [Grafana plugin Catalog](https://grafana.com/grafana/plugins/).
```

Comment:
Nit: it's best to rewrite any sentence with a semicolon.

Reply:
Will go with your original suggestion then 👍